app/views/admin/claims/index.html.erb

```diff
@@ -1,5 +1,6 @@
+<a href="/admin" class="govuk-back-link">Back</a>
```

Comment:
If you update the `routes.rb` file as described in the comment below, you can change this to:
```suggestion
<%= link_to "Back", admin_root_path, class: "govuk-back-link" %>
```